packages/esm-service-queues-app/src/patient-queue-header/patient-queue-header.component.tsx

```diff
@@ -14,7 +14,10 @@ import {
 } from '../helpers/helpers';
 import styles from './patient-queue-header.scss';
 
-const PatientQueueHeader: React.FC<{ title?: string }> = ({ title }) => {
+const PatientQueueHeader: React.FC<{ title?: string; hideLocationDropdown?: boolean }> = ({
```

Comment:
I would think it's generally better to have `showLocationDropdown`, rather than `hideLocationDropdown`.

Reply:
Yes, I agree, but since this is already implemented, to not modify the existing implementation, I made a new variable that decides whether to hide or not.

I'll make the change!

Reply:
Not sure I totally follow @vasharma05 .  What is already implemented?